lib/service/service.go

```diff
@@ -66,21 +66,46 @@ var log = logrus.WithFields(logrus.Fields{
 })
 
 const (
-	// ProxyReverseTunnelServerEvent is generated supervisor when proxy
-	// has initialized reverse tunnel server
-	ProxyReverseTunnelServerEvent = "ProxyReverseTunnelServer"
-	// ProxyWebServerEvent is generated supervisor when proxy
-	// has initialized web tunnel server
-	ProxyWebServerEvent = "ProxyWebServer"
-	// ProxyIdentityEvent is generated by supervisor when proxy's identity has been initialized
+	// AuthIdentityEvent is generated when the Auth Servers identity has been
+	// initialized in the backend.
+	AuthIdentityEvent = "AuthIdentity"
+
+	// ProxyIdentityEvent is generated by the supervisor when the proxy's
+	// identity has been registered with the Auth Server.
 	ProxyIdentityEvent = "ProxyIdentity"
-	// SSHIdentityEvent is generated when node's identity has been received
+
+	// SSHIdentityEvent is generated when node's identity has been registered
+	// with the Auth Server.
 	SSHIdentityEvent = "SSHIdentity"
-	// TeleportExitEvent is generated when someone is askign Teleport Process to close
-	// all listening sockets and exit
+
+	// DELETE IN: 2.6.0
+	// AuthServerSSHReady is generated when the Auth Server has initialized the
+	// HTTP-over-SSH endpoint and is ready to start accepting connections.
+	AuthServerSSHReady = "AuthServerReady"
```

Comment:
remvoving this will break backwards compatibility with Telekube that relies on this event, so I would keep it - (just remove DELETE IN comment line)

Reply:
This is a new event. Does Telekube rely on `ProxyReverseTunnelServerEvent` or `ProxyWebServerEvent`? Those are the two events I renamed.

Reply:
can you check? I don't recall

Reply:
Shouldn't be an issue, event is still emitted with a different name: https://github.com/gravitational/gravity/blob/master/lib/process/process.go#L911

Since it's all one process, shouldn't be any backward compatibility issues.